torch/csrc/Event.h

```diff
@@ -7,15 +7,15 @@
 struct TORCH_API THPEvent {
   PyObject_HEAD c10::Event event;
 };
-extern PyObject* THPEventClass;
+TORCH_API extern PyTypeObject* THPEventClass;
```

Comment:
You add this to our public API because you need it on a third party c++ code?

Reply:
Yes, I think the out-of-tree backend needs this to support linear inheritance.